openstack/compute/v2/extensions/hypervisors/results.go

```diff
@@ -123,6 +144,12 @@ type Hypervisor struct {
 	// Service is the service this hypervisor represents.
 	Service Service `json:"service"`
 
+	// Servers is a list of Server object.
+	// The requires microversion 2.53 or later.
+	// This field has become mandatory in microversion 2.75.
```

Comment:
I recommend we remove this line. "mandatory" doesn't make sense here, even if it's in the OpenStack documentation. This is a field that is being returned from OpenStack, so there's no action that a developer or user can do that is "mandatory".